### PR TITLE
Update RNEosEcc.podspec

### PR DESCRIPTION
**Pull request by Turtus**
new xcode fails with empty homepage

### CODE REVIEW

**Review by stevenans9859 — APPROVED** [review]

File: ios/RNEosEcc.podspec [hunk 1]
```diff
@@ -6,7 +6,7 @@ Pod::Spec.new do |s|
   s.description  = <<-DESC
                   RNEosEcc
                    DESC
-  s.homepage     = ""
+  s.homepage     = "https://fioprotocol.io/"
```
Comment (ericbutz) [inline ios/RNEosEcc.podspec:9]: Change to: "https://github.com/EvaCoop/react-native-eos"

This copies what is at: https://github.com/EvaCoop/react-native-eos/blob/master/RNEosEcc.podspec

Comment (andreyvEze) [reply ios/RNEosEcc.podspec:9]: @ericbutz the develop branch already has the value (`https://github.com/author/RNEosEcc.git`) - https://github.com/fioprotocol/fiosdk_typescript/blob/develop/ios/RNEosEcc.podspec#L9

**Comment by andreyvEze** [comment]
https://github.com/fioprotocol/fiosdk_typescript/pull/120

File: ios/RNEosEcc.podspec (hunk 1 above)
Comment (andreyvEze) [reply ios/RNEosEcc.podspec:9]: new PR https://github.com/fioprotocol/fiosdk_typescript/pull/120

**Pull request closed without merging**